onnxruntime/test/mlas/bench/bench_sqnbitgemm.cpp

```diff
@@ -84,57 +84,3 @@ BENCHMARK(SQNBITGEMM<4, 128, false>)->Apply(GemmSizeProducts)->UseRealTime();
 BENCHMARK(SQNBITGEMM<4, 128, true>)->Apply(GemmSizeProducts)->UseRealTime();
 BENCHMARK(SQNBITGEMM<4, 256, false>)->Apply(GemmSizeProducts)->UseRealTime();
 BENCHMARK(SQNBITGEMM<4, 256, true>)->Apply(GemmSizeProducts)->UseRealTime();
-
```

Comment:
Has a benchmark been established for NSSQNBitsGemmBatchPackedB following its removal?

Reply:
Is there any suggestion for a contrib_ops' benchmark way?

Reply:
Why you can't just use the original MlasNBitsGemmPackBSize, MlasNBitsGemmPackB, MlasSQNBitsGemmBatchPackedB (NSNBitsGemmPackBSize, NSNBitsGemmPackB, NSSQNBitsGemmBatchPackedB)?

Reply:
The source files are moved to contrib_ops as suggested. It's no longer part of MLAS. It can't use these MLAS APIs.

Reply:
The benchmark method can be added in the next PR, as an executable target of neural_speed itself.